shared/dataflow/codeql/dataflow/test/ProvenancePathGraph.qll

```diff
@@ -102,17 +123,6 @@ module TestPostProcessing {
 
     private module Models = TranslateModels<interpretModelForTest/2, provenance/1>;
 
-    private newtype TModelRow = TMkModelRow(int r, string model) { Models::models(r, model) }
```

Comment:
Maybe keep this as a backstop against the case when `interpretModelForTest` fails to be functional.

Reply:
Or just apply `concat` to force `interpretModelForTest` to be functional.